catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -247,6 +248,42 @@
         <property name="endpointUtil" ref="endpointUtil" />
     </bean>
 
+    <bean id="jsonRpc" class="org.codice.ddf.catalog.ui.ws.JsonRpc">
+        <argument>
+            <map>
+                <entry key="query" value-ref="queryApplication" />
+            </map>
+        </argument>
+    </bean>
+
+    <reference id="manager" interface="ddf.security.service.SecurityManager" />
+
+    <bean id="socketThreadFactory"
+          class="org.codice.ddf.platform.util.StandardThreadFactoryBuilder"
+          factory-method="newThreadFactory">
+        <argument value="socketThreadFactory"/>
+    </bean>
+
+    <ext:property-placeholder/>
```

Comment:
For documentation's sake
http://aries.apache.org/modules/blueprintweb.html#configuring-blueprint-through-properties